components/experimental/src/transliterate/compile/mod.rs

```diff
@@ -120,6 +120,18 @@ impl RuleCollection {
         }
     }
 
+    /// Add transliteration ID aliases without registering a source.
+    pub fn register_aliases<'a>(
+        &mut self,
+        id: &icu_locale_core::Locale,
+        aliases: impl IntoIterator<Item = &'a str>,
+    ) {
+        for alias in aliases.into_iter() {
+            self.id_mapping
+                .insert(alias.to_ascii_lowercase(), id.clone());
+        }
```

Comment:
nit: use `extend` instead of repeated `insert`

Reply:
Done